README.md

```diff
@@ -175,6 +175,14 @@ services:
 The Docker image is "best effort" and intended for advanced users; questions
 about running Docker are outside the scope of Neolink.
 
+If you use a UDP only camera, such as a battery camera you will need to either
+use `--net=host` or setup a [macvlan](https://docs.docker.com/network/macvlan/)
+for the docker image that supports UDP broadcast.
+This is because UDP requires that udp broadcast messages are transmitted across
```

Comment:
Is it possible to detect this situation and yell? I suppose not, but I'm hopeful because I suspect it will save support requests.

Reply:
Also, nit, would suggest using the non-technical term "battery camera" first, as in:

> If you use a battery-powered camera (or other UDP-only camera)...

Reply:
Done